src/pages/Shared/FeedCard/FeedCard.js

```diff
@@ -0,0 +1,141 @@
+        expand,
+        ...other
+    } = props;
+    return <IconButton {...other}/>;
+})(({theme, expand}) => ({
+    transform: !expand ? "rotate(0deg)" : "rotate(180deg)",
+    marginLeft: "auto",
+    transition: theme.transitions.create("transform", {duration: theme.transitions.duration.shortest})
+}));
+
+export const FeedCard = (props) => {
+    const [expanded, setExpanded] = React.useState(false);
+
+    const handleExpandClick = () => {
+        setExpanded(!expanded);
+    };
+
+    const label = {
+        inputProps: {
+            "aria-label": "Checkbox demo"
+        }
+    };
+
+
+    return (
+        <>
+        {/* <CssBaseline/> */}
+        <Card 
+            sx={
+                {marginTop: "10px"}
+        }>
+            <CardHeader avatar={<Avatar
+                    sx={
+                    {bgcolor: red[500]}}
+                    aria-label="recipe">
+                    R</Avatar>}
+                action={
+                    <IconButton
+                aria-label="settings"><MoreVertIcon/></IconButton>
+                }
+                title={<Typography variant="h6" color="text.tertiary">Shrimp and Chorizo Paella</Typography>}
+                subheader={<Typography color="text.tertiary">September 14, 2016</Typography>}
+                className="card-header"
+                color="primary"
+                id="card-header"
+            /> 
+            <CardMedia component="img"
+                // height="20%"
+                image={
+                    props.image
+                }
+                alt="Paella dish"
+                id="feedcard-img"
+                className="feedcard-img"/>
+            <CardContent>
+                <Typography variant="body2" color="text.tertiary">
+                    This impressive paella is a perfect party dish and a fun meal to cook
+                              together with your guests. Add 1 cup of frozen peas along with the
+                              mussels, if you like.
+                </Typography>
+            </CardContent>
+            <Divider/>
+            <CardActions disableSpacing>
+                <IconButton aria-label="add to favorites">
+                    <Checkbox {...label}
+                        icon={<FavoriteBorder/>}
+                        checkedIcon={<Favorite/>}
+                      />
+                </IconButton>
+                <IconButton aria-label="share">
+                    <ShareIcon/>
+                </IconButton>
+                <ExpandMore expand={expanded}
+                    onClick={handleExpandClick}
+                    aria-expanded={expanded}
+                    aria-label="show more">
+                    <ExpandMoreIcon/>
+                </ExpandMore>
+            </CardActions>
+            <Collapse in={expanded}
+                timeout="auto"
+                unmountOnExit>
+                {/* <CardContent>
+                    <Typography paragraph>Method:</Typography>
```

Comment:
comments can be removed if not required

Reply:
these are commented for future perspective